app/models/katello/content_view.rb

```diff
@@ -469,7 +469,7 @@ def check_remove_from_environment!(env)
       }
 
       dependencies.each do |key, name|
-        if (models = self.association(key).scoped.in_environment(env)).any?
+        if (models = self.association(key).scope.in_environment(env)).any?
```

Comment:
not sure if there is a reason that isn't changed to ```where(nil)``` or ```all```. I don't see a reason not to change to ```.all```